test/lightning/invocation_test.exs

```diff
@@ -749,11 +806,238 @@ defmodule Lightning.InvocationTest do
       insert(:log_line,
         run: run,
         step: step,
-        message: "Sadio Mane is playing in Senegal",
+        message: "Sadio Mane is playing for Senegal",
+        timestamp: Timex.now()
+      )
+
+      insert(:log_line,
+        run: run,
+        step: step,
+        message: "Bukayo Saka is playing for England",
         timestamp: Timex.now()
       )
 
-      assert Lightning.Invocation.search_workorders(
+      %{
+        project: project,
+        dataclip: dataclip,
+        workorder: workorder,
+        run: run,
+        step: step
+      }
+    end
+
+    @tag skip: "Ooops. We don't support this yet."
```

Comment:
this is something i'd like to support in the future, but it would be a progressive enhancement from where are now.